packages/framework/tinylicious-client/src/test/TinyliciousClient.spec.ts

```diff
@@ -229,7 +235,8 @@ describe("TinyliciousClient", () => {
 
         const map1 = createFluidContainer.initialObjects.map1 as SharedMap;
         map1.set("newpair-id", newPair.handle);
-        const obj = await map1.get("newpair-id").get();
+        const hdl = await map1.get("newpair-id") as IFluidHandle;
+        const obj = await hdl.get() as TestDataObject;
```

Comment:
So is this a bug??

Reply:
Not really, no. The problem is that our maps and directories are untyped, so you have to cast the returned value. There's an issue somewhere tracking better type support in maps (maybe generics).

This case is worse because you have to cast the handle too. Without the types everything works fine, so it's not a runtime bug, but without this change I can't satisfy the no-unsafe rule.

Reply:
@microsoft/fluid-cr-dds Do you have any opinions about the patterns we encourage for accessing DDSes behind handles? Is this the right way to do it?